src/Shmuelie.WinRTServer/ComServer.cs

```diff
@@ -113,6 +133,10 @@ public unsafe bool RegisterClassFactory(BaseClassFactory factory)
         {
             throw new ArgumentNullException(nameof(factory));
         }
+        if (comWrappers is null)
+        {
+            throw new ArgumentNullException(nameof(comWrappers));
```

Comment:
You're on .NET 8 now, replace all of these with `ArgumentNullException.ThrowIfNull(comWrappers)` in the whole repo 😄